### PR TITLE
Create heroku.yml

### PR DESCRIPTION
**Pull request by telegrambotdev**
heroku docker support added

### CODE REVIEW

File: heroku.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+build:
+  docker:
+    web: Dockerfile
+run:
+  web: bash start.sh
```
Comment (Rudloff) [inline heroku.yml:5]: Since this needs start.sh, shouldn't it be in the same PR as https://github.com/Rudloff/alltube/pull/317?

**Comment by Rudloff** [comment]
Is this is merged, does it mean deployments will start using the Docker image?
The Docker image is not well maintained, so I would not want to use it for https://alltubedownload.net/.

File: heroku.yml (hunk 1 above)
Comment (telegrambotdev) [reply heroku.yml:5]: > Since this needs start.sh, shouldn't it be in the same PR as #317?

yes.thats right

**Comment by telegrambotdev** [comment]
> Is this is merged, does it mean deployments will start using the Docker image?
> The Docker image is not well maintained, so I would not want to use it for https://alltubedownload.net/.

lol no.The code will not prompt to deploy as a Dockerfile,its just a fix.you can try it your own.

**Comment by telegrambotdev** [comment]
> Is this is merged, does it mean deployments will start using the Docker image?
> The Docker image is not well maintained, so I would not want to use it for https://alltubedownload.net/.

yup.i understand your concern.so shall i close the PR ?

**Comment by Rudloff** [comment]
I tried to push your branch on my Heroku instance and it did not automatically try to use the Docker image.
According to the doc, it seems to only do so if the stack is `container` (my stack is `heroku-18`).
So I guess we can keep the `heroku.yml` file.

**Comment by telegrambotdev** [comment]
Thank you.

**Pull request closed without merging**